docs/examples/ssh/SSH_BITBUCKET.md

```diff
@@ -0,0 +1,24 @@
+# SSH service in Bitbucket on Kubernetes
+
+In addition to providing a service on HTTP(S), Bitbucket also allows remote Git operations over SSH connections. By default, Kubernetes Ingress controllers only work for HTTP connections, but some ingress controllers also support TCP connections.
+
+You can provide SSH access by creating a separate Kubernetes `LoadBalancer` service, but you can do this only if you don’t need your deployment to have the SSH service available on the same DNS name as the HTTP service.
```

Comment:
When I am thinking about this, could we structure the document like:

## Purpose/goal/...

In addition to providing...

## Prerequisites

1. You can run your SSH service on a different domain name
2. You have external-dns (?optional/recommended)

## Steps
1. Enable SSH service
2. (Optional) Add DNS annotation
3. Apply new values

Reply:
yep that also looks good, much better structure